controllers/likedVideo.controller.js

```diff
@@ -0,0 +1,86 @@
+const User = require("../models/user-model");
+const LikedVideo = require("../models/likedVideo-model");
+
+const getLikedVideos = async (req, res) => {
+  const likedVideos = await LikedVideo.find({});
+  res.json({ success: true, likedVideos });
+};
+
+const findUserLikedVideo = async (req, res, next, userId) => {
+  try {
+    let user = await User.findOne({ _id: userId });
```

Comment:
`findById` can be used here too

Reply:
Yes can  be used but I have read that findOne and findById handles undefined differently so I chose findOne

Reply:
Yes they handle differently . But `findOne(undefined)` will return an arbitrary document from the users collection.
[Reference](https://mongoosejs.com/docs/api.html#model_Model.findOne)